src/libraries/Microsoft.NETCore.Platforms/tests/Microsoft.NETCore.Platforms.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);net472</TargetFrameworks>
+    <IgnoreForCI Condition="'$(TargetOS)' == 'Browser'">true</IgnoreForCI>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\Microsoft.NETCore.Platforms.csproj" />
+    <!-- Workaround NuGet promoting this to a ProjectReference -->
+    <PackageReference Include="System.Memory" Condition="'$(TargetFramework)' == 'net472'" Version="$(SystemMemoryVersion)" />
```

Comment:
Can you please elaborate? Why does NuGet add a P2P for System.Memory?

Reply:
NuGet tried to restore System.Memory.csproj for `net472` without this and restore failed.  I believe it's because it's in the closure for the project added by the test targets.  I think it's the same issue as https://github.com/NuGet/Home/issues/9354.